flowclient/flowclient/aggregates.py

```diff
@@ -0,0 +1,1007 @@
+        label are returned.
+    labels : dict of dicts
+        A dictionary whose keys are the label names and the values geojson-style shapes,
+        specified hour of day, and day of week score, with hour of day score on the x-axis
+        and day of week score on the y-axis, where all scores are real numbers in the range [-1.0, +1.0]
+    aggregation_unit : str
+        Unit of aggregation, e.g. "admin3"
+    tower_day_of_week_scores : dict
+        A dictionary mapping days of the week ("monday", "tuesday" etc.) to numerical scores in the range [-1.0, +1.0].
+
+        Each of a subscriber's interactions with a tower is given a score for the day of the week it took place on. For
+        example, passing {"monday":1.0, "tuesday":0, "wednesday":0, "thursday":0, "friday":0, "saturday":0, "sunday":0}
+        would score any interaction taking place on a monday 1, and 0 on all other days. So a subscriber who made two calls
+        on a monday, and received one sms on tuesday, all from the same tower would have a final score of 0.666 for that
+        tower.
+    tower_hour_of_day_scores : list of float
+        A length 24 list containing numerical scores in the range [-1.0, +1.0], where the first entry is midnight.
+        Each of a subscriber's interactions with a tower is given a score for the hour of the day it took place in. For
+        example, if the first entry of this list was 1, and all others were zero, each interaction the subscriber had
+        that used a tower at midnight would receive a score of 1. If the subscriber used a particular tower twice, once
+        at midnight, and once at noon, the final hour score for that tower would be 0.5.
+    tower_cluster_radius : float
+        When constructing clusters, towers will be considered for inclusion in a cluster only if they are within this
+        number of km from the current cluster centroid. Hence, large values here will tend to produce clusters containing
+        more towers, and fewer clusters.
+    tower_cluster_call_threshold : int
+        Exclude towers from a subscriber's clusters if they have been used on less than this number of days.
+    subscriber_subset : dict or None
+        Subset of subscribers to retrieve modal locations for. Must be None
+        (= all subscribers) or a dictionary with the specification of a
+        subset query.
+
+    Returns
+    -------
+    Query
+        Meaningful locations aggregate query
+
+    References
+    ----------
+    .. [1] S. Isaacman et al., "Identifying Important Places in People's Lives from Cellular Network Data", International Conference on Pervasive Computing (2011), pp 133-151.
+    .. [2] Zagatti, Guilherme Augusto, et al. "A trip to work: Estimation of origin and destination of commuting patterns in the main metropolitan regions of Haiti using CDR." Development Engineering 3 (2018): 133-165.
+
+    Notes
+    -----
+    Does not return any value below 15.
+    """
+    return Query(
```

Comment:
Can we change the name of the class? I think I'd prefer APIRequest or something like that. My rationale is that it disambiguates it from the `Query` class in FM.

Reply:
Is `APIQuery` a fair compromise? To my mind, `APIRequest` sounds like it would encompass all API requests, including things like `get_available_dates`.

Reply:
Fine with that :+1: